netutils/nng/Makefile

```diff
@@ -0,0 +1,152 @@
+CSRCS += $(NNG_SRCDIR)/core/thread.c
+CSRCS += $(NNG_SRCDIR)/core/timer.c
+CSRCS += $(NNG_SRCDIR)/core/url.c
+
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_alloc.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_atomic.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_clock.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_debug.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_file.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_ipcconn.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_ipcdial.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_ipclisten.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_pipe.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_resolv_gai.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_sockaddr.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_tcpconn.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_tcpdial.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_tcplisten.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_thread.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_udp.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_pollq_epoll.c
+CSRCS += $(NNG_SRCDIR)/platform/posix/posix_rand_getrandom.c
+#CSRCS += $(NNG_SRCDIR)/platform/posix/posix_rand_urandom.c
+
+CSRCS += $(NNG_SRCDIR)/sp/transport.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/bus0/bus.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/pair0/pair.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/pair1/pair.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/pair1/pair1_poly.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/pipeline0/pull.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/pipeline0/push.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/pubsub0/pub.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/pubsub0/sub.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/pubsub0/xsub.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/reqrep0/rep.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/reqrep0/req.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/reqrep0/xrep.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/reqrep0/xreq.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/survey0/respond.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/survey0/survey.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/survey0/xrespond.c
+CSRCS += $(NNG_SRCDIR)/sp/protocol/survey0/xsurvey.c
+CSRCS += $(NNG_SRCDIR)/sp/transport/inproc/inproc.c
+CSRCS += $(NNG_SRCDIR)/sp/transport/ipc/ipc.c
+CSRCS += $(NNG_SRCDIR)/sp/transport/tcp/tcp.c
+CSRCS += $(NNG_SRCDIR)/sp/transport/tls/tls.c
+CSRCS += $(NNG_SRCDIR)/sp/transport/ws/websocket.c
+
+CSRCS += $(NNG_SRCDIR)/supplemental/base64/base64.c
+CSRCS += $(NNG_SRCDIR)/supplemental/http/http_chunk.c
+CSRCS += $(NNG_SRCDIR)/supplemental/http/http_client.c
+CSRCS += $(NNG_SRCDIR)/supplemental/http/http_conn.c
+CSRCS += $(NNG_SRCDIR)/supplemental/http/http_msg.c
+CSRCS += $(NNG_SRCDIR)/supplemental/http/http_public.c
+CSRCS += $(NNG_SRCDIR)/supplemental/http/http_schemes.c
+CSRCS += $(NNG_SRCDIR)/supplemental/http/http_server.c
+CSRCS += $(NNG_SRCDIR)/supplemental/sha1/sha1.c
+CSRCS += $(NNG_SRCDIR)/supplemental/tls/tls_common.c
+#CSRCS += $(NNG_SRCDIR)/supplemental/tls/mbedtls/tls.c
+#CSRCS += $(NNG_SRCDIR)/supplemental/util/options.c
```

Comment:
why skip mbedtls

Reply:
> why skip mbedtls

I think, something with compiling. And I didn't need secure sockets. Will investigate further.

Reply:
ok